### ISSUE TITLE
Cannot install ECL CLI

### ISSUE THREAD

**Issue by MaKu119** [nttcom/eclcli#36]
Error happens when I try to install ECL CLI as below. How can I solve it?

$ pip install eclcli
Downloading/unpacking eclcli
  Downloading eclcli-1.1.3.tar.gz (470kB): 470kB downloaded
  Running setup.py (path:/tmp/pip_build_ubuntu/eclcli/setup.py) egg_info for package eclcli
    error in setup command: ("EntryPoint must be in 'name=module:attrs [extras]' format", 'dns_zone_create = eclcli.dns.v2.zone: CreateZone')
    Complete output from command python setup.py egg_info:
    error in setup command: ("EntryPoint must be in 'name=module:attrs [extras]' format", 'dns_zone_create = eclcli.dns.v2.zone: CreateZone')

----------------------------------------
Cleaning up...
Command python setup.py egg_info failed with error code 1 in /tmp/pip_build_ubuntu/eclcli
Storing debug log for failure in /home/ubuntu/.pip/pip.log

**Comment by sb3f-ryo-sato** [comment]
I have same error messages on CentOS 7.

```
[root@centos7 ~]# pip install eclcli
Collecting eclcli
  Downloading eclcli-1.1.3.tar.gz (470kB)
    76% |?????????????????????????       | 358kB 3.6MB/s eta 0:00:0    78% |?????????????????????????       | 368kB 3.3MB/s eta 0:00:0    80% |??????????????????????????      | 378kB 3    100% |????????????????????????????????| 471kB 1.4MB/s
    Complete output from command python setup.py egg_info:

    Installed /tmp/pip-build-ezmkh4/eclcli/pbr-3.1.1-py2.7.egg
    running egg_info
    creating pip-egg-info/eclcli.egg-info
    writing requirements to pip-egg-info/eclcli.egg-info/requires.txt
    writing pip-egg-info/eclcli.egg-info/PKG-INFO
    writing top-level names to pip-egg-info/eclcli.egg-info/top_level.txt
    writing dependency_links to pip-egg-info/eclcli.egg-info/dependency_links.txt
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-build-ezmkh4/eclcli/setup.py", line 10, in <module>
        pbr=True)
      File "/usr/lib64/python2.7/distutils/core.py", line 152, in setup
        dist.run_commands()
      File "/usr/lib64/python2.7/distutils/dist.py", line 953, in run_commands
        self.run_command(cmd)
      File "/usr/lib64/python2.7/distutils/dist.py", line 972, in run_command
        cmd_obj.run()
      File "/usr/lib/python2.7/site-packages/setuptools/command/egg_info.py", line 178, in run
        writer(self, ep.name, os.path.join(self.egg_info,ep.name))
      File "/usr/lib/python2.7/site-packages/setuptools/command/egg_info.py", line 467, in write_entries
        contents = EntryPoint.parse_group(section, contents)
      File "/usr/lib/python2.7/site-packages/pkg_resources.py", line 2324, in parse_group
        ep = cls.parse(line, dist)
      File "/usr/lib/python2.7/site-packages/pkg_resources.py", line 2303, in parse
        src
    ValueError: ("EntryPoint must be in 'name=module:attrs [extras]' format", 'dns_zone_create = eclcli.dns.v2.zone: CreateZone')

    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-build-ezmkh4/eclcli/
[root@centos7 ~]#
```

**Comment by ghost** [comment]
@MaKu119 @sb3f-ryo-sato Thank you for reporting the issue. We are trying to find out the cause now.

**Comment by ghost** [comment]
@MaKu119 @sb3f-ryo-sato 
This issue occurs if version of `setuptools`  is old and `eclcli` <= 1.1.3.
We believe it's resolved by Pull Request #37. Please install the latest version of eclcli 1.1.4 by `pip install eclcli`.

You can also try `pip install -U setuptools` to solve it.

**Comment by sb3f-ryo-sato** [comment]
@htysh Thanks a lot.
I could install eclcli 1.1.4 in CentOS 7 with setuptools 0.9.8.
The setuptools is installed by rpm.
```
$ rpm -q python-setuptools
python-setuptools-0.9.8-4.el7.noarch

$ pip show setuptools
Name: setuptools
Version: 0.9.8
Summary: Easily download, build, install, upgrade, and uninstall Python packages
Home-page: https://pypi.python.org/pypi/setuptools
Author: Python Packaging Authority
Author-email: distutils-sig@python.org
License: PSF or ZPL
Location: /usr/lib/python2.7/site-packages
Requires:
```
